sdk_lib/sdk_entry.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -eu
```

Comment:
I don't think this is a good idea - if there is any issue with the entry script then this would prevent you from entering the container ever again, potentially losing a lot of (unrelated) work.

A bug in this script could be fixed from within the container; erroring out before establishing a shell would prevent you from entering the container though.

Reply:
Ok, it means that we miss these error in CI, though